src/components/theme/View/View.jsx

```diff
@@ -229,6 +230,27 @@ class View extends Component {
           )}
           <title>{this.props.content.title}</title>
           <meta name="description" content={this.props.content.description} />
+
+          <meta property="og:title" content={this.props.content.title} />
+          {/* <meta property="og:type" content="" /> */}
+          <meta
+            property="og:url"
+            content={toPublicURL(this.props.content['@id'])}
+          />
+          {this.props.content.image?.scales?.large?.download && (
```

Comment:
Where does the image come from? Isn't that just empty for most content types?

Reply:
If present (it's the lead image) we have the image here
We could consider a fallback but I don't know what we could use

Reply:
Can we make a configurable utility?
```
export default applyConfig(config) {
    const {extractOpenGraph} = config.settings;

    config.settings.extractOpenGraph = (data) => {
        if (data['@type'] === 'Contact']) {
            return {'og:image': data['portrait']........}
        }
       return extractOpenGraph(data);
    }
}
```